SPECS/kernel/config_aarch64

```diff
@@ -9236,6 +9238,7 @@ CONFIG_IIO=y
 # CONFIG_AD7923 is not set
 # CONFIG_AD7949 is not set
 # CONFIG_AD799X is not set
+# CONFIG_AD9467 is not set
```

Comment:
result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/184b2967d578a62af32b97f9d02be869fc54080a